tests/test_assets/gene_info/input/pharos_classes_good_input.csv

```diff
@@ -0,0 +1,20 @@
+ensembl_gene_id,uniprot_id,hgnc_symbol,pharos_class
+ENSG00000000005,Q9H2S6,TNMD,Tbio
```

Comment:
Based on your expected output files, it looks like no gene in the test data set is missing a pharos class. I think it would be a good idea to test that case, by simply deleting a few genes from the pharos class input that exist in the gene_info dataset as a whole.

Reply:
There are many several genes not featured in the `pharos_classes_good_input.csv` file that result in `"druggability":null` values in the output. But I can also just remove a value from the CSV and see what happens :)

Reply:
Oh I see, I missed the places in the output where it's null because I was only looking at the diff. My bad, no changes needed here!

Reply:
No worries, I actually have no idea what happens if the pharos_class column value is missing in the input, so you inspired me to check haha :)